File: class2/main.py


DATA = [10,12,20,50,30,20]
N = 5
LOC=0
k=1
print(DATA)
ITEM=int(input("Enter the item to search for:"))

for k in range(N):
	if(DATA[k]==ITEM):
		LOC = k+1
		break;

if LOC == 0:
	print("ITEM not in array data")
else:
	
	print("Loc is found in location",LOC)

File: class2/main.c
#include <stdio.h>

int main() {
    int data[] = {10, 20, 30, 40, 50};
    int n = 4;
    int item;
    int loc = 0;

    printf("Enter the item to search for: ");
    scanf("%d", &item);

    for (int k = 0; k < n; k++) {
        if (data[k] == item) {
            loc = k;
            break;
        }
    }

    if(loc == 0){
    	printf("Item is not found \n ");

    }else {
    	printf("Item is found at %d \n ",loc);
    }
}
